From c4e0a7568131137cc098dbad39b2fe0b602950bf Mon Sep 17 00:00:00 2001
From: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Date: Fri, 20 May 2022 08:27:02 +0300
Subject: [PATCH 42/50] fdts: s32cc: Correct RDC simple_bus_reg warning

This corrects dtc warning:
Warning (simple_bus_reg): /soc/ethernet@46080000: simple-bus unit
address format error, expected "46000000"

Issue: ALB-8823
Upstream-Status: Pending 

Signed-off-by: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 fdts/s32g.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fdts/s32g.dtsi b/fdts/s32g.dtsi
index 4ae22ac19..9fa29a86c 100644
--- a/fdts/s32g.dtsi
+++ b/fdts/s32g.dtsi
@@ -162,7 +162,7 @@
 				<&plat_clks S32G_CLK_ACCEL_PLL_PHI1>;
 		};
 
-		pfe: ethernet@46080000 {
+		pfe: ethernet@46000000 {
 			compatible = "fsl,s32g274a-pfe";
 			reg = <0x0 0x46000000 0x0 0x1000000>;
 			clocks = <&clks S32G_SCMI_CLK_PFE_PE>,
-- 
2.17.1

